lib/engine/game/g_18_uruguay/step/destination_bonus.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Engine
+  module Game
+    module G18Uruguay
+      module Step
+        class DestinationToken < Engine::Step::Base
```

Comment:
Renaming to match name of the file and the actual function of the step

```suggestion
        class DestinationBonus< Engine::Step::Base
```